spec/serializers/api/v2/green_lanes/goods_nomenclature_serializer_spec.rb

```diff
@@ -1,19 +1,23 @@
 RSpec.describe Api::V2::GreenLanes::GoodsNomenclatureSerializer do
   subject(:serialized) do
-    described_class.new(gn_presenter, include: %w[applicable_measures applicable_category_assessments applicable_category_assessments.geographical_area]).serializable_hash
+    assessments_with_measures = [[category_assessment, [first_measure]]]
+
+    gn_presenter = \
+      Api::V2::GreenLanes::GoodsNomenclaturePresenter.new(subheading,
+                                                          assessments_with_measures)
```

Comment:
a bit of strange indentation here :smile_cat:

Reply:
Really? I've seen it lots in code bases and is actually what the govuk'ised rubocop recommends to use?